credentials/credentials.go

```diff
@@ -100,13 +120,17 @@ func Get(helper Helper, reader io.Reader, writer io.Writer) error {
 	}
 
 	serverURL := strings.TrimSpace(buffer.String())
+	if len(serverURL) == 0 {
```

Comment:
got it.  But potentially, the username could create an invalid [`Credentials` struct in the code below](https://github.com/docker/docker-credential-helpers/pull/62/files#diff-9311ff8f32d06930be1eabeb53b517d6R134) right?  I would prefer to validate so that any invalid `Credentials` struct is rejected, in case the semantics of the `Get` operation change.

This can be a followup since I acknowledge the importance of this fix and that this is not an issue at present - I can help with the PR.